src/h5web/vis-packs/core/matrix/MappedMatrixVis.tsx

```diff
@@ -28,7 +29,9 @@ function MappedMatrixVis(props: Props): ReactElement {
     [dimMapping, dims]
   );
 
-  const baseArray = useBaseArray(value, slicedDims);
+  const baseArray = useBaseArray(value, slicedDims) as ndarray<
+    string | number | boolean
+  >;
```

Comment:
Same problem of incompatible types between `ndarray<unknown>` and `ndarray<string | number | boolean>`...

I could only fix it with a cast but perhaps you have a better idea...